petvax-app/petvax-app/src/app/components/footer/footer.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="container">
+    <div id = "column1">
```

Comment:
You should use a class if you need to use the same selector more than once within a page or a site. While an ID is specific to a single element, classes can be assigned to multiple elements on a page or throughout the website. In this case, ID is better for single element columns because these have different CSS properties.